pkg/api/metadata.go

```diff
@@ -81,6 +82,11 @@ func (m *Metadata) Basename() string {
 	return fmt.Sprintf("%s.yaml", basename)
 }
 
+// JobFilePath returns the file path for the jobs of the supplied suffix type
+func (m *Metadata) JobFilePath(suffix string) string {
+	return filepath.Join(m.Org, m.Repo, fmt.Sprintf("%s-%s-%s-%s.yaml", m.Org, m.Repo, m.Branch, suffix))
```

Comment:
The job generation in `WriteToDir` would have had to also compute this somehow, can we make sure it's DRY with this new method?

Reply:
It is doing a bit extra as it creates all the directories (if they don't exist) too. It doesn't use the Metadata struct, but rather uses strings for `org` and `repo`. It could probably be refactored to use this new method, but it wouldn't be entirely trivial. Thoughts?